features/step_definitions/forum/post_steps.rb

```diff
@@ -73,7 +73,7 @@
 end
 
 Given "{user} posted {string} {today} under {topic}" do |user, body, time, topic|
-  Timecop.travel(Chronic.parse(time)) do
+  no_nest_travel_to(Chronic.parse(time)) do
```

Comment:
:eyes:  this method has been removed, from what I can see

Reply:
Ouch! fixed...